### DISCUSSION TITLE
Responsive width variant(?) not working

### DISCUSSION THREAD

**Discussion by Lilja** [tailwindlabs/tailwindcss#10221] [Help] [answered]
I currently have an element with `sm:w-2/12`. A width that seems to be defined in the width page of the documentation, however If I throw a `sm:` modifier in between, it doesn't seem to work...

**Comment by wongjn** [comment, marked as answer]
Some things to check:
- Whether the `sm:w-2/12` appears in the compiled CSS and the rule applies to the class in the DOM. If it does, then it could be that your expectations of what the CSS rule should be doing may be misaligned to what it actually does.
- Whether you are looking at the webpage with a viewport wider than `640px` (assuming the default `screens.sm` configuration). 
- Tailwind has compiled since you added the class.
- The file that the class appears in is covered by the [`content` file globs](https://tailwindcss.com/docs/content-configuration).
- The class appears as a full, unbroken string. In other words, [string interpolation or concatenation is **not** used to construct the class name string](https://tailwindcss.com/docs/content-configuration#dynamic-class-names).
- `w-2/12` is a valid width utility class. In other words, any Tailwind configuration customizations has not removed `2/12` from being a `width` value.
- `sm:` is a valid variant. In other words, any Tailwind configuration customizations has not removed `sm` from being a key in the [`screens`](https://tailwindcss.com/docs/screens) configuration.
